worker/rhpv3.go

```diff
@@ -674,7 +674,7 @@ func padBandwidth(pt rhpv3.HostPriceTable, rc rhpv3.ResourceCost) rhpv3.Resource
 	}
 	minPacketSize := uint64(1460)
 	minIngress := pt.UploadBandwidthCost.Mul64(minPacketSize)
-	minEgress := pt.DownloadBandwidthCost.Mul64(3 * minPacketSize)
+	minEgress := pt.DownloadBandwidthCost.Mul64(3*minPacketSize + responseLeeway)
```

Comment:
Not sure where the 3 comes from here, was introduced by #502. Could've just made it `5` but decided to add the `responseLeeway` we use in `RPCReadSector`.. can be anything as long as it's higher. I was thinking of using a 4kib min. on both upload and download bandwidth... just didn't want to get rid of the packet size padding.